src_c/_sdl2/controller.c

```diff
@@ -26,11 +26,19 @@ static PyObject *
 controller_module_init(PyObject *module, PyObject *_null)
 {
     if (!SDL_WasInit(SDL_INIT_GAMECONTROLLER)) {
+#if SDL_VERSION_ATLEAST(3, 0, 0)
+        if (SDL_InitSubSystem(SDL_INIT_GAMECONTROLLER) == SDL_FALSE) {
```

Comment:
```suggestion
        if (!SDL_InitSubSystem(SDL_INIT_GAMECONTROLLER)) {
```
minor nit: if its a boolean we don't need `==` checks, just to enfore style consistency. Same applies to other instances in this file